gtk4/src/subclass/editable.rs

```diff
@@ -282,6 +362,24 @@ unsafe extern "C" fn editable_get_text<T: EditableImpl>(
         .to_glib_full()
 }
 
+static EDITABLE_GET_DELEGATE_QUARK: Lazy<Quark> =
+    Lazy::new(|| Quark::from_string("gtk-rs-subclass-editable-get-delegate"));
+
+unsafe extern "C" fn editable_get_delegate<T: EditableImpl>(
+    editable: *mut ffi::GtkEditable,
+) -> *mut ffi::GtkEditable {
+    let instance = &*(editable as *mut T::Instance);
+    let imp = instance.get_impl();
+
+    let wrap = from_glib_borrow::<_, Editable>(editable);
+
+    let delegate = imp.get_delegate(wrap.unsafe_cast_ref());
+
+    let delegateptr = delegate.to_glib_full();
+    wrap.set_qdata(*EDITABLE_GET_DELEGATE_QUARK, delegateptr);
+    delegateptr
```

Comment:
This is a transfer none, i believe this the right way to handle it? :)

Reply:
You probably want to ensure that it's always the same pointer too, and you want to make sure the pointer is unreffed when the object is destroyed. So why not store `delegate.clone()`?